x/posts/legacy/v0.8.0/migrate_test.go

```diff
@@ -0,0 +1,206 @@
+			},
+			v060.PostReaction{
+				Owner:     postCreator,
+				Shortcode: ":my_house:",
+				Value:     "https://myHouse.png",
+			},
+		}},
+		RegisteredReactions: []v040.Reaction{
+			{
+				ShortCode: ":my_house:",
+				Value:     "https://myHouse.png",
+				Subspace:  subspace,
+				Creator:   postCreator,
+			},
+		},
+	}
+
+	expected := v060.GenesisState{
+		Posts: []v040.Post{
+			{
+				PostID:         parentID,
+				ParentID:       "",
+				Message:        "Message",
+				AllowsComments: true,
+				Subspace:       "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e",
+				OptionalData:   map[string]string{},
+				Created:        parentCreationTime,
+				LastEdited:     time.Time{},
+				Creator:        parentPostCreator,
+				Medias:         []v040.PostMedia{{URI: "https://uri.com", MimeType: "text/plain"}},
+			},
+			{
+				PostID:         postID,
+				ParentID:       parentID,
+				Message:        "Message",
+				AllowsComments: true,
+				Subspace:       "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e",
+				OptionalData:   map[string]string{},
+				Created:        postCreationTime,
+				LastEdited:     time.Time{},
+				Creator:        postCreator,
+				Medias:         []v040.PostMedia{{URI: "https://uri.com", MimeType: "text/plain"}},
+			},
+		},
+		UsersPollAnswers: map[string][]v040.UserAnswer{string(postID): {v040.UserAnswer{
+			Answers: []v040.AnswerID{1, 2},
+			User:    postCreator,
+		}}},
+		PostReactions: map[string][]v060.PostReaction{string(postID): {
+			v060.PostReaction{
+				Owner:     postCreator,
+				Shortcode: ":fire:",
+				Value:     "🔥",
+			},
+			v060.PostReaction{
+				Owner:     postCreator,
+				Shortcode: ":my_house:",
+				Value:     "https://myHouse.png",
+			},
+		}},
+		RegisteredReactions: []v040.Reaction{
+			{
+				ShortCode: ":my_house:",
+				Value:     "https://myHouse.png",
+				Subspace:  subspace,
+				Creator:   postCreator,
+			},
+		},
+	}
+
+	migrated := v080.Migrate(v060GenState)
+
+	// Check for posts
+	require.Len(t, migrated.Posts, len(expected.Posts))
+	for index, post := range migrated.Posts {
+		require.Equal(t, expected.Posts[index].PostID, post.PostID)
+		require.Equal(t, expected.Posts[index].ParentID, post.ParentID)
+	}
+
+	// Check for users poll answers
+	require.Len(t, migrated.UsersPollAnswers, len(expected.UsersPollAnswers))
+	for key := range expected.UsersPollAnswers {
+		require.Equal(t, expected.UsersPollAnswers[key], migrated.UsersPollAnswers[key])
+	}
+
+	// Check for posts reactions
+	require.Len(t, migrated.PostReactions, len(expected.PostReactions))
+	for key := range expected.PostReactions {
+		require.Equal(t, expected.PostReactions[key], migrated.PostReactions[key])
+	}
+
+	require.Len(t, migrated.RegisteredReactions, len(expected.RegisteredReactions))
+	for index, reaction := range migrated.RegisteredReactions {
+		require.Equal(t, expected.RegisteredReactions[index], reaction)
+	}
+
+}
+
+func TestMigrate060(t *testing.T) {
```

Comment:
I think this reference to `060` as well as all the ones below should be `080` instead.

Reply:
I was thinking that it's fine because (as we did with the previous versions) this is the test that migrate `v0.6.0` to the `v0.8.0`

Reply:
@bragaz Yeah the method is fine, it's just the names that are bugging me :joy: We are inside the `v0.8.0` folder and the method is called `TestMigrate060` instead as `TestMigrate080` :joy: